core/bug_api.php

```diff
@@ -544,6 +546,52 @@ function create() {
 		history_log_event_direct( $this->id, 'status', $t_original_status, $t_status );
 		history_log_event_direct( $this->id, 'handler_id', 0, $this->handler_id );
 
+		# Now that the issue is added process the @ mentions
+		$t_all_mentioned_user_ids = array();
```

Comment:
I'm not convinced this is the right place to process the mention tags
as BugData->create() is the core method for creating a bug, this may arise to triggering mentions in more situation than those intended.
Example of a debatible situation: Cloning an issue
1) Does it have to trigger mentions contained in the texts? and/or in the notes?
2) The user that is informed as mentioner is "auth_get_current_user_id", this means that if user A clones an issue where B did a comment mentioning C, a new mention will be triggerd as "user A mentions B" (i tested that it does so)

This login probably fits better in the processing of submitted text, right after user interaction, where current user is still  the "mentioner", and this way, mentions are triggered only at the time of submitting.